compression/lpf.py

```diff
@@ -0,0 +1,73 @@
+
+from string_indexing import lcp, suffix_array
```

Comment:
Please delete the blank line at the top (or even better: run make check to see if there are any style issues)

Reply:
Ack. The `make check` did not catch this (maybe it needs an explicit rule)